utils/dockerd/files/etc/config/dockerd

```diff
@@ -21,6 +21,13 @@ config globals 'globals'
 #	list registry_mirrors 'https://<my-docker-mirror-host>'
 #	list registry_mirrors 'https://hub.docker.com'
 
+# If your organization uses a proxy server to connect to the internet, you may need to configure the proxy.
+# See https://docs.docker.com/engine/daemon/proxy/ for more details
+config proxies 'proxies'
+#	option http_proxy 'http://proxy.example.com:3128'
+#	option https_proxy 'https://proxy.example.com:3129'
+#	option http_proxy '.example.com,localhost,127.0.0.0/8,169.254.0.0/16,172.16.0.0/12,192.168.0.0/16,10.0.0.0/8'
```

Comment:
As a general rule, if there isn't a general reasonable default, I prefer to have just enough to explain how it's used.
Primarily to avoid people submitting PRs to change these to something that works better for them specifically.
For example, people could use the standard http/s ports (80/443) from their proxies, or even have servers only accessible from non-public IP addresses, behind a proxy. And there isn't even an IPv6 example! :stuck_out_tongue_winking_eye: 

what are your thoughts on using the ones from the docker documentation?
```json
  "proxies": {
    "http-proxy": "http://proxy.example.com:80",
    "https-proxy": "https://proxy.example.com:443",
    "no-proxy": "*.test.example.com,.example.org"
  },
```
I figure if someone does complain it removes the authority of the matter from us.

Reply:
I agree with you, just give examples for typical use cases, and at the same time to explain how to use.
Actually these examples are from https://docs.docker.com/engine/daemon/proxy/

```
{
  "proxies": {
    "http-proxy": "http://proxy.example.com:3128",
    "https-proxy": "https://proxy.example.com:3129",
    "no-proxy": "*.test.example.com,.example.org,127.0.0.0/8"
  }
}
```

For the additional `127.0.0.0/8,169.254.0.0/16,172.16.0.0/12,192.168.0.0/16,10.0.0.0/8`, it looks very long, it is actually for private IPv4 addresses defined in [RFC 1918](https://tools.ietf.org/html/rfc1918), I think it is a reasonable default for no_proxy. But yes, there is no IPv6 examples, I am not familar with that.

Reply:
Ah, I didn't see that link. 
As for the additional IP addresses, I did recognise them, I was just attempting to provide other equally valid options that could also be there to point out that there are plenty of reasonable options. So maybe we should keep it succinct like the alternative set you posted even. 
For more reason, some people may have a remote LAN via a proxy and may want some of these typically non-routed addresses to go via the proxy. So, in some, maybe rare, cases, these aren't a reasonable default.

Reply:
Yes, there is no universally reasonable default. So shall I change it as following?

```json
  "proxies": {
    "http-proxy": "http://proxy.example.com:3128",
    "https-proxy": "https://proxy.example.com:3129",
    "no-proxy": "*.test.example.com,.example.org"
  },
```

I prefer to use port numbers other than 80 and 443 to distinguish between http proxies and the well known http web servers.

Reply:
perfect!